src/algebra/star/pi.lean

```diff
@@ -62,3 +62,7 @@ lemma star_sum_elim {I J α : Type*} (x : I → α) (y : J → α) [has_star α]
 by { ext x, cases x; simp }
 
 end function
+
+instance _root_.function.star_module (α β : Type*) [has_star α] [has_star β] [has_smul α β]
```

Comment:
Can you add the usual pi instance not firing disclaimer here?

Reply:
Like this https://github.com/leanprover-community/mathlib/blob/ac3910d7f80f7d2d8abaf73af3ba751f60e2d009/src/algebra/module/pi.lean#L63-L72 ?

Reply:
@eric-wieser, don't we have a library note that we can reference?

Reply:
@eric-wieser Did you have any comment to make on this, or shall I resolve it?